pages/content/amp-dev/documentation/guides-and-tutorials/develop/interactivity_guide/custom_experiences.md

```diff
@@ -0,0 +1,284 @@
+---
+formats:
+  - websites
+$title: Building custom interactive experiences
```

Comment:
can we name the file personalized experiences - that will SEO better

Reply:
you got it ;)